adapter/mssql/mssql.go

```diff
@@ -14,6 +14,725 @@ import (
 
 const indent = "    "
 
+// names from: https://docs.microsoft.com/en-us/sql/t-sql/language-elements/reserved-keywords-transact-sql?view=sql-server-ver15
+var reservedKeywordsArray = [...]string{
```

Comment:
Isn't just escaping every identifier sufficient? If not, could we possibly reuse this table? https://github.com/k0kubun/sqldef/blob/888f344ebf388d5224eb3eb69a9299e4a751ae17/sqlparser/token.go#L94

Reply:
I think escaping everything is better. I didn't because I read only small parts of source code so was afraid breaking something.

Reply:
And now I knew that token keywords map are root of my original issue.